MathLib/LinAlg/LinAlg.h

```diff
@@ -206,6 +210,16 @@ namespace LinAlg
 
 // Vector
 
+/**
+    Set local accessible vector in order to get entries. Call this
+    before call operator[] or get(...) of x.
+    The function only has computation if DDC is appied,
+    e.g. parallel computing. Up to now, Eigen vector is not used for
+    global vectors in parallel computing. Therefore this function is
+    empty in the current status.
+*/
```

Comment:
Maybe you could mention that the purpose of this overloaded function is to fulfil the lin alg interface (is this correct?). For this reason, it has an empty implementation.